packages/commerce-sdk-react/src/constant.ts

```diff
@@ -14,6 +14,17 @@ export const IFRAME_HOST_ALLOW_LIST = Object.freeze([
     'https://runtime-admin-preview.mobify-storefront.com'
 ])
 
+// We hard code /mobify/proxy here as we don't want commerce-sdk-react to have a dependency on pwa-kit-runtime
+export const PROXY_PATH = '/mobify/proxy'
+
+// We hard code /mobify/bundle/development here as we don't want commerce-sdk-react to have a dependency on pwa-kit-runtime
+export const LOCAL_BUNDLE_PATH = '/mobify/bundle/development'
+
+// We hard code /mobify/slas/private here as we don't want either of the following:
+// * commerce-sdk-react to have a dependency on pwa-kit-runtime
+// * /mobify/slas/private to be exposed for configuration via AuthConfig
+export const SLAS_PRIVATE_PROXY_PATH = '/mobify/slas/private'
```

Comment:
Currently I am defining `/mobify` paths in 2 places. We could get rid of this duplication if we have commerce-sdk-react depend on pwa-kit-runtime but I don't know if that is a good idea.